### DISCUSSION TITLE
Baseline Source Files to Match Precompiled Firmware

### DISCUSSION THREAD

**Discussion by GeoStork** [Jyers/Marlin#178] [Q&A] [answered]
NOOB question....   I'm using the precompiled BLTouch 5x5 firmware.... if i want to have the exact source copy so I can compile myself with tweaks. Are their preconfigured "Configuration .h" and "Configuration_adv.h" that I need/should copy to the Marlin directory so I'm close to the source that would compile to the config I"m running?   I want to make sure I have the correct source baseline that matches my precompiled firmware.   Hope this makes sense.  Thanks.

**Comment by clintiepoo** [comment, marked as answer]
There is a folder called “Configuration Files” in the root of this branch. It has everything.

**Reply by funkybside** [reply]
I came here for the same reason and found this question.     The "Configuration Files" folder does not have everything, as far as I can tell.   For example, in precompiled there is a version for E3V2_BLTouch_5x5_HS_v4.2.2.bin.    There does not appear to be any BLTouch_5x5_HS file in the configuration folder.

**Reply by clintiepoo** [reply]
Agree - I think you'll just want to look at line 1379 in Configuration.h. Change 3 to 5

```
#if EITHER(AUTO_BED_LEVELING_LINEAR, AUTO_BED_LEVELING_BILINEAR)

  // Set the number of grid points per dimension.
  #define GRID_MAX_POINTS_X 3
  #define GRID_MAX_POINTS_Y GRID_MAX_POINTS_X
```
